api/s2n.h

```diff
@@ -421,6 +421,97 @@ extern int s2n_connection_set_client_auth_type(struct s2n_connection *conn, s2n_
 S2N_API
 extern int s2n_connection_get_client_cert_chain(struct s2n_connection *conn, uint8_t **der_cert_chain_out, uint32_t *cert_chain_len);
 
+/**
+ * Sets the initial number of session tickets to send after a handshake.
+ *
+ * @param config A pointer to the config object.
```

Comment:
To be precise use:

- S2N config and  S2N connection terminology or 
- `s2n_config` and `s2n_connection` as used below

Reply:
Idk about s2n_config/s2n_connection. Isn't the exact type covered by the function signature?

And why would adding "S2N" be more precise?

Reply:
I added the exact type, as this is how we are formatting the comments on other APIs.